docs/UserGuide.md

```diff
@@ -32,80 +32,173 @@
 > **Command Format**
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
-> * The order of parameters is fixed.
+> * The order of parameters is fixed, with optional parameters being the exception.
+> * Parameters `START_DATE` and `END_DATE` has to be in dd-mm-yyyy format (e.g. 23-10-2016).
```

Comment:
think if you can make use of some libraries to relax this restriction? i believe you guys can do it :-) 
take a look at [natty](http://natty.joestelmach.com/) or [prettytime](http://www.ocpsoft.org/prettytime/)